ui/package.json

```diff
@@ -15,8 +15,11 @@
     "test": "jest"
   },
   "dependencies": {
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
+    "@types/react-router-dom": "^5.1.5",
+    "react": ">= 16.8.0",
+    "react-dom": ">= 16.8.0",
+    "react-is": ">= 16.8.0",
```

Comment:
why `react-is`?

Reply:
Hmm, we didn't intentionally add it. But it may(?) be useful https://www.npmjs.com/package/react-is

Reply:
We can always re-add later on, we should just remove it if we are not using rn.